lib/chef/provider/package/freebsd/base.rb

```diff
@@ -58,7 +58,8 @@ def port_dir(port)
 
           def makefile_variable_value(variable, dir = nil)
             options = dir ? { cwd: dir } : {}
-            options.merge!(env: nil, returns: [0, 1])
+            options[:env] = nil
+            options[:returns] = [0, 1]
             make_v = shell_out!("make", "-V", variable, **options)
             make_v.exitstatus == 0 ? make_v.stdout.strip.split($OUTPUT_RECORD_SEPARATOR).first : nil # $\ is the line separator, i.e. newline.
```

Comment:
```suggestion
            make_v.exitstatus == 0 ? make_v.stdout.strip.split($OUTPUT_RECORD_SEPARATOR).first : nil
```

I know it's not your responsibility to fix everything, but this comment doesn't make sense anymore (assuming the code used $\` originally).